scss/civicrm/campaign/pages/_campaign.scss

```diff
@@ -17,4 +17,62 @@
       }
     }
   }
+
+  .CRM_Campaign_Form_Task_Interview {
+    .float-right {
+      margin-bottom: 16px;
+      margin-right: 14px;
+    }
+
+    .dataTables_length {
+      margin-left: 20px;
+    }
+
+    .dataTables_info {
+      padding-top: 11px;
+    }
+
+    .crm-copy-fields {
+      border-bottom: 1px solid $gray-light;
+      border-top: 1px solid $gray-light;
+    }
+
+    .civireport-criteria {
+      .form-layout-compressed {
+        @extend %crm-table;
```

Comment:
is this `@extend` really necessary?

Reply:
yes, or I need to write all css for that table again. with targeting each part. so, better make it extended.